src/main/java/io/lettuce/core/CompositeRedisCommandListener.java

```diff
@@ -0,0 +1,37 @@
+package io.lettuce.core;
+
+import io.lettuce.core.models.events.CommandFailedEvent;
+import io.lettuce.core.models.events.CommandStartedEvent;
+import io.lettuce.core.models.events.CommandSucceededEvent;
+
+import java.util.List;
+
+/**
+ * @author sokomishalov
+ */
+public class CompositeRedisCommandListener implements RedisCommandListener {
+    private final List<RedisCommandListener> listeners;
+
+    public CompositeRedisCommandListener(List<RedisCommandListener> listeners) {
+        this.listeners = listeners;
+    }
+
+    @Override
+    public <K, V, T> void commandStarted(CommandStartedEvent<K, V, T> event) {
+        listeners.forEach(l -> l.commandStarted(event));
```

Comment:
Using good old for-loops avoids capturing lambdas.

Reply:
fixed